### PR TITLE
Veeam Cloud Connect Usage Report

### PR DESCRIPTION
**Pull request by mycloudrevolution**
This Script will Report Cloud Connect Tenant Statistics

### CODE REVIEW

File: VCC-UsageReport/VCC-UsageReport.ps1 [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+[cmdletbinding()]
+param(
+    [Parameter(Position=0, Mandatory=$false)]
+    	[String] $Server = "VeeamEM.lan.local",
+	[Parameter(Position=1, Mandatory=$false)]
+		[Boolean] $HTTPS = $True,
+	[Parameter(Position=2, Mandatory=$false)]
+		[String] $Port = "9398",
+	[Parameter(Position=3, Mandatory=$false)]
+		[String] $Authentication = "Vk9QXHN2Yy12cm6tY2MwMTp2XltKNUNiS2dlIUp6dkQxbkdiZnky"
```
Comment (poulpreben) [inline VCC-UsageReport/VCC-UsageReport.ps1]: You should remove this authentication line. base64 is perfectly reversible, so everyone will be able to see you service account password.

File: VCC-UsageReport/VCC-UsageReport.ps1 [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            return true;
+        }
+    }
+"@
+[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
+#endregion
+
+#region: Switch Http/s
+if ($HTTPS -eq $True) {$Proto = "https"} else {$Proto = "http"}
+#endregion
```
Comment (poulpreben) [inline VCC-UsageReport/VCC-UsageReport.ps1]: If proto = http, default port is 9399 by default. Suggest adding a check here if port == 9398 && proto == http, and if yes, change port to 9399 as well? Just an idea :)

**Comment by poulpreben** [comment]
@mycloudrevolution Nice work! I've added some suggestions and one critical fix to be implemented before merging this pull. Thanks for your collaboration!

File: VCC-UsageReport/VCC-UsageReport.ps1 (hunk 1 above)
Comment (mycloudrevolution) [inline VCC-UsageReport/VCC-UsageReport.ps1]: Have you tried to Decode it?
It should only be an exapmple...

Comment (poulpreben) [inline VCC-UsageReport/VCC-UsageReport.ps1]: I have indeed, and it does look like a legit username and password. 

You can try here: https://www.base64decode.org/

Comment (mycloudrevolution) [inline VCC-UsageReport/VCC-UsageReport.ps1]: It is a modified Base64 Hash. User ad Password do not match.

But I completely remove it and add another commit.

Comment (poulpreben) [inline VCC-UsageReport/VCC-UsageReport.ps1]: That is good to know! When removing the string, please add a comment about how to build the base64 string.

**Comment by poulpreben** [comment]
Looks good! Only one last request:

As per convention, please move usage information to a `README.md` file. You may see an example here: https://github.com/VeeamHub/powershell/blob/master/Surebackup-SQLCheck/README.md

**Pull request merged**